components/organisms/ContributorCard/contributor-card.tsx

```diff
@@ -69,7 +69,7 @@ const ContributorCard = ({ className, contributor, topic, repositories, range }:
         <div className="flex justify-center w-full">
           <button
             onClick={() => setShowPRs((prevState) => !prevState)}
-            className="w-full py-1 transition bg-white border rounded-lg border-light-slate-6 hover:bg-light-slate-1"
+            className="w-full py-1 transition bg-white border rounded-lg border-light-slate-6 hover:bg-light-slate-3"
```

Comment:
These changes aren't related to the fix. Please open a separate issue to discuss these changes.

```suggestion
            className="w-full py-1 transition bg-white border rounded-lg border-light-slate-6 hover:bg-light-slate-1"
```